java-codegen/src/main/java/org/opensearch/client/codegen/openapi/MimeType.java

```diff
@@ -14,8 +14,11 @@
 import org.opensearch.client.codegen.utils.Strings;
 
 public enum MimeType {
+    Cbor("application/cbor"),
```

Comment:
Smile seems to be missing (we do support it)

Reply:
Note that the spec doesn't have it yet, https://github.com/opensearch-project/opensearch-api-specification/issues/372.